components/admin_console/team_channel_settings/team/details/team_details.jsx

```diff
@@ -361,57 +479,44 @@ export default class TeamDetails extends React.PureComponent {
 
                 <div className='admin-console__wrapper'>
                     <div className='admin-console__content'>
-                        <RemoveConfirmModal
-                            amount={usersToRemoveCount}
-                            inChannel={false}
-                            show={showRemoveConfirmation}
-                            onCancel={this.hideRemoveUsersModal}
-                            onConfirm={this.handleSubmit}
-                        />
                         <TeamProfile
                             team={team}
-                        />
-
-                        <TeamModes
-                            allAllowedChecked={allAllowedChecked}
-                            allowedDomainsChecked={allowedDomainsChecked}
-                            allowedDomains={allowedDomains}
-                            syncChecked={syncChecked}
-                            onToggle={this.setToggles}
+                            onToggleArchive={this.onToggleArchive}
+                            isArchived={isLocalArchived}
                             isDisabled={this.props.isDisabled}
                         />
-
-                        <TeamGroups
-                            syncChecked={syncChecked}
-                            team={team}
-                            groups={groups}
-                            removedGroups={removedGroups}
-                            totalGroups={totalGroups}
-                            onAddCallback={this.handleGroupChange}
-                            onGroupRemoved={this.handleGroupRemoved}
-                            setNewGroupRole={this.setNewGroupRole}
-                            isDisabled={this.props.isDisabled}
+                        <ConfirmModal
+                            show={showArchiveConfirmModal}
+                            title={
+                                <FormattedMessage
+                                    id='admin.team_settings.team_detail.archive_confirm.title'
+                                    defaultMessage='Save and Archive Team'
+                                />
+                            }
+                            message={
+                                <FormattedMessage
+                                    id='admin.team_settings.team_detail.archive_confirm.message'
+                                    defaultMessage='Saving will archive the team and make its contents inaccessible for all users. Are you sure you wish to save and archive this team?'
+                                />
+                            }
+                            confirmButtonText={
+                                <FormattedMessage
+                                    id='admin.team_settings.team_detail.archive_confirm.button'
+                                    defaultMessage='Save and Archive Team'
```

Comment:
It seems odd that the save button and modal title both say "Save and Archive Team". Maybe the button should just say "Archive"?